RawVideo/Package.appxmanifest

```diff
@@ -17,7 +17,7 @@
   <mp:PhoneIdentity PhoneProductId="646c0233-13f7-4ea3-b02e-703b86c51a27" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
 
```

Comment:
You should also take this opportunity to upgrade the ACS SDK to the latest of 1.4.0.0. Right now it still uses 1.0.0.0, way too old.

Reply:
i dont know why that version got installed thanks for the catch